Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "23.1.0-rc.2"
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" "23.1.1"
```

Comment:
I think it's okay for Carthage. For SPM and CocoaPods this wouldn't work because of Maps and Navigation Native versions mismatch in Common.